.github/workflows/coverage_report.yaml

```
@@ -0,0 +1,36 @@
+name: coverage_report
+
+on:
+  schedule:
+    - cron: '0 0 * * 5'
+
+jobs:
+  coverage_report:
+    name: coverage_report
+    runs-on: [self-hosted,Linux]
```

According to the convention in https://talk.nervos.org/t/rfc-ckb-improve-ci-workflows/6053

```suggestion
    runs-on: self-hosted-coverage-report-ubuntu-??
```

For `??`, I'm not sure which version we are using on the self runners.

---

our self runner version is 20.04